include/igraph_cycles.h

```diff
@@ -0,0 +1,27 @@
+
+#ifndef IGRAPH_CYCLES_H
+#define IGRAPH_CYCLES_H
+
+#include "igraph_decls.h"
+#include "igraph_interface.h"
+#include "igraph_types.h"
+#include "igraph_vector_list.h"
+
+__BEGIN_DECLS
+
+IGRAPH_EXPORT igraph_error_t igraph_fundamental_cycles(
+        const igraph_t *graph,
+        igraph_integer_t start_vid,
+        igraph_integer_t cutoff,
+        igraph_vector_int_list_t *result);
```

Comment:
We haven't agreed on the policy for the placement of the `result` argument. I vaguely recall that in most functions the ordering is: the graph comes first, then the output arguments, then all the input arguments. It is my subjective perception only, though. My point is only that we should probably try to stick to whatever is predominant now in the library.

Reply:
I have no preference. As I recall, @vtraag suggested the output arg to come last, that's why it's last here. Let me know where you want it and I can easily refactor it.

Reply:
I'll make a quick survey in the functions of the library to see where `res` and `result` are typically placed and then get back to you.

Reply:
See the result on our Slack channel

Reply:
Done.